Automattic/woocommerce-payments
# Update multi currency settings UX

Fixes #2335

#### Changes proposed in this Pull Request

- Remove none and add 1 to zero-decimal rounding options
- Fix decimal currency rounding defaulting to none instead of 1
- Do not render decimals for zero-decimal currencies preview









#### Testing instructions





*Decimal currency*

* Add a new decimal currency
* Open its settings and check that the preview has decimal places
* Check that the default rounding option is 1.00

*Zero-decimal currency*

* Open the currency settings for a zero-decimal currency
* Check that the minimum rounding option is 1
* Check that the preview doesn't have any decimal places


-------------------
